governance/polls/Add GUNIV3DAIUSDCFT with 0.01% Fee Tier as a new Vault Type - November 29, 2021.md

```diff
@@ -0,0 +1,87 @@
+
+## Review
+
+This poll allows the MakerDAO governance community to signal their support or opposition to adding GUNIV3DAIUSDCFT (Gelato DAI-USDC) with 0.01% Fee Tier as a new vault type in the Maker Protocol with the parameters below:
+
+### Initial Collateral Parameters
+
+* Underlying Collateral: GUNI-DAI/USDC
+* [Stability Fee](https://community-development.makerdao.com/en/learn/governance/param-stability-fee): 1%
+* [Liquidation Ratio](https://community-development.makerdao.com/en/learn/governance/param-liquidation-ratio): 105%
+* [Debt Ceiling (`line`)](https://community-development.makerdao.com/en/learn/governance/param-debt-ceiling): 10 million DAI
+* [Debt Floor (`dust`)](https://community-development.makerdao.com/en/learn/governance/param-debt-floor): 10,000 DAI
+* [Target Available Debt (`gap`)](https://makerdao.world/en/learn/governance/module-dciam): 10 million DAI
+* [Ceiling Increase Cooldown (`ttl`)](https://makerdao.world/en/learn/governance/module-dciam): 8 hours
+
+
+### Liquidation 2.0 Parameters
+
+* [Auction Price Function (`calc`)](https://community-development.makerdao.com/en/learn/governance/param-auction-price-function): Stairstep Exponential
+* [Price Change Multiplier (`cut`)](https://community-development.makerdao.com/en/learn/governance/param-auction-price-function): 0.999
+* [Price Change Interval (`step`)](https://community-development.makerdao.com/en/learn/governance/param-auction-price-function): 120 seconds
+* [Auction Price Multiplier (`buf`)](https://community-development.makerdao.com/en/learn/governance/param-auction-price-multiplier): 1.05
+* [Liquidation Penalty (`chop`)](https://community-development.makerdao.com/en/learn/governance/param-liquidation-penalty): 13%
+
+*Liquidations are disabled for this collateral type. The risk for GUNIV3DAIUSDCFT is primarily from USDC tail risks, and so it's impossible to generate appropriate risk parameters from previous data.*
+
+**Limits**
+
+* [Local Liquidation Limit (`ilk.hole`)](https://community-development.makerdao.com/en/learn/governance/param-local-liquidation-limit): 5 million DAI
+* [Maximum Auction Drawdown (`cusp`)](https://community-development.makerdao.com/en/learn/governance/param-max-auction-drawdown): 0.9
+* [Maximum Auction Duration (`tail`)](https://community-development.makerdao.com/en/learn/governance/param-max-auction-duration): 220 minutes
+* [Breaker Price Tolerance (`tolerance`)](https://community-development.makerdao.com/en/learn/governance/param-breaker-price-tolerance): 0.95
+
+**Incentives**
+
+* [Proportional Kick Incentive (`chip`)](https://community-development.makerdao.com/en/learn/governance/param-proportional-kick-incentive): 0.10%
+* [Flat Kick Incentive (`tip`)](https://community-development.makerdao.com/en/learn/governance/param-flat-kick-incentive): 300 DAI
+
+Please review the following forum threads containing information about GUNIV3DAIUSDCFT (Gelato) to inform your position before voting.
```

Comment:
PE have confirmed the ilk will be called GUNIV3DAIUSDC2-A :)

Reply:
Fixed!